.github/workflows/node.js.yml

```diff
@@ -83,3 +96,15 @@ jobs:
           token: ${{ secrets.CODECOV_TOKEN }}
           directory: ./coverage
           flags: unit
+
+      - name: Install Playwright dependencies
+        run: PLAYWRIGHT_BROWSERS_PATH=0 npx playwright install --with-deps
+      - name: Run Playwright tests
+        run: PLAYWRIGHT_BROWSERS_PATH=0 npx playwright test
+      - name: Upload Playwright report
```

Comment:
Can you put a blank line above each of the `name` blocks so it visually breaks the steps up a bit?